api/policy/policy.go

```diff
@@ -216,6 +216,69 @@ type DecisionRequest struct {
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
 }
 
+// GetSettings calls the GET decision-settings API of policy-service.
+func (c Client) GetSettings(ownerID string, context string) (interface{}, error) {
+	path := fmt.Sprintf("%s/api/v1/owner/%s/context/%s/decision/settings", c.serverUrl, ownerID, context)
+	req, err := http.NewRequest("GET", path, nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to construct request: %v", err)
+	}
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		var payload httpError
+		if err := json.NewDecoder(resp.Body).Decode(&payload); err != nil {
+			return nil, fmt.Errorf("unexpected status-code: %d", resp.StatusCode)
+		}
+		return nil, fmt.Errorf("unexpected status-code: %d - %s", resp.StatusCode, payload.Error)
+	}
+
+	var body interface{}
+	if err := json.NewDecoder(resp.Body).Decode(&body); err != nil {
+		return nil, fmt.Errorf("failed to decode response body: %v", err)
+	}
+
+	return body, nil
+}
+
+// DecisionSettings represents a request to Policy-Service to configure decision settings.
+type DecisionSettings struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
We don't need a `*bool` from the client side, since we know we MUST send a boolean value and not null. On the server we had to make the distinction between false and absent value

Reply:
This is to have a tri-state bool. In case, the user wants to modify other settings, but keep enabled as-is, client won't send the enabled flag in the body.

Reply:
We should update our Rest Verbage to Patch then before releasing

Reply:
I was about to say the exact same thing. Sure, I'll do it.